classroom.rb

```diff
@@ -0,0 +1,60 @@
+# Represents a classroom with attributes such as label and students
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    students << student
+    student.classroom = self
+  end
+end
+
+# Represents a student with attributes name and classroom.
+class Student
+  attr_accessor :name, :classroom
+
+  def initialize(name)
+    @name = name
+    @classroom = nil
+  end
+end
+
+# Represents a book with attributes such as title, author, and rentals.
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] When you create a `book.rb` file and put this `Book` class in it, kindly create the `add_rental` method in the Book class. Remember that the `Book` and `Person` classes have a **_many-to-many_** relationship through the `Rental` class. This can be successfully implemented with the creation of the `@rentals` array and the `add_rental` methods in both the `Book` and `Person` classes. This would enable their instances to keep records of all the rentals that belong to that instance. 👍 😎 

To successfully implement the relationship between both classes. The `add_rental` method should create a new instance of the `Rental` class. 💻 🖱️ 

#### Hint:

```ruby
def add_rental
  Rental.new(date, self, person)
end
```

Reply:
@Gambit142 done